mail/common/src/main/java/com/fsck/k9/mail/internet/MimeHeaderEncoder.kt

```diff
@@ -17,12 +19,15 @@ object MimeHeaderEncoder {
     }
 
     private fun exceedsRecommendedLineLength(name: String, value: String): Boolean {
-        return name.length + COLON_PLUS_SPACE_LENGTH + value.length > RECOMMENDED_MAX_LINE_LENGTH
+        return name.headerPrefixLength + value.length > RECOMMENDED_MAX_LINE_LENGTH
     }
 
     private fun charactersNeedEncoding(text: String): Boolean {
         return text.any { !it.isVChar() && !it.isWspOrCrlf() }
     }
 
     private const val COLON_PLUS_SPACE_LENGTH = 2
+
+    private val String.headerPrefixLength: Int
+        get() = length + COLON_PLUS_SPACE_LENGTH
```

Comment:
This could help with the above issue:

```suggestion
    /**
    * Calculates the length of the header prefix, which includes the length of the name and a constant space for colon and space.
    */
    private val String.headerPrefixLength: Int
        get() = length + COLON_PLUS_SPACE_LENGTH
```

or replace it with this:

```suggestion
    /**
    * Calculates the length of the header prefix, which includes the length of the name and a constant space for colon and space.
    */
    private fun headerPrefixLength(name: String): Int {
        return name.length + COLON_PLUS_SPACE_LENGTH
    }
```

Reply:
I don't think a comment like this adds any value. It's a 1:1 translation from the code to English.